src/checks/hyper-training-31.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  description: 'A maxed IV (31) cannot be hypertrained.',
+  field: 'iv',
+  test (pkmn) {
+    return !(
+      pkmn.ivHp === 31 && pkmn.hyperTrainedHp ||
```

Comment:
So just to clarify - should it be done like the other test where it's split in their separate cases?